release-notes.md

```diff
@@ -221,4 +229,4 @@
 * `.code.text` ; mapping ; Moved mapping of FurtherSpecificationProblemName from `.code.extension:furtherSpecificationProblemName.value[x]`. ; ZIBFHIR-145
 * `.bodySite` ; cardinality ; Changed max cardinality from 1 to *. ; ZIB-1433
 * `.bodySite` ; terminology ; Binding strength changed from required to extensible. ; ZIBFHIR-113
-* `.note.text` ; textual ; Changed definition of Comment. ; ZIB-1610
\ No newline at end of file
+* `.note.text` ; textual ; Changed definition of Comment. ; ZIB-1610
```

Comment:
Snelle check: gaat dit goed? Zit dit niet al in de main.

Reply:
Ik begrijp je opmerking niet? Dit gaat over zib-Problem